README.md

```diff
@@ -280,6 +280,22 @@ render plain: 'Ruby!'
 ...
 ```
 
+* <a name="symbols-over-numeric-status-codes"></a>
+  Prefer corresponding symbols to numeric status codes. They are meaningful and do not look like "magic" numbers for less known status codes.
+<sup>[[link](#symbols-over-numeric-status-codes)]</sup>
+
+```Ruby
+# not desired
+...
+render status: 500
+...
+
+# good
+...
+render status: :forbidden
```

Comment:
Should it be rails documentation? (http://guides.rubyonrails.org/layouts_and_rendering.html) this [gist](https://gist.github.com/mlanett/a31c340b132ddefa9cca) or the link I provided in merge request' description?